rules/allied-infantry.yaml

```diff
@@ -106,26 +108,38 @@ e1:
 		Weapon: M60
 		UpgradeTypes: eliteweapon
 		UpgradeMaxEnabledLevel: 0
-		UpgradeMaxAcceptedLevel: 1
+		UpgradeMaxAcceptedLevel: 0
 	Armament@elite:
 		Weapon: M60E
-		UpgradeTypes: eliteweapon
+		UpgradeTypes: eliteweapon, undeployed
```

Comment:
This is the problem, such combinations aren't supported AFAIK. Remove `undeployed` from here.